index.html

```diff
@@ -3,12 +3,48 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/styles.css" />
     <title>Currency converter</title>
 </head>
 <body>
+    <header>
+        <h1>Welcome to New Currency Converter</h1>
+    </header>
     <main>
+        <form>
+            <fieldset class="frame">
+                <legend>From</legend>
+            <label class="lbl_form" for="amount_input">Amount</label>
+            <input id="amount_input" type="number" width="25px" placeholder="1">
```

Comment:
you should set also a default `value` here e.g: 1. Otherwise it will throw NaN result if you immediately click on covert button